app/Console/Commands/CacheDelegateVoterCounts.php

```diff
@@ -31,8 +32,24 @@ final class CacheDelegateVoterCounts extends Command
      */
     public function handle(): void
     {
-        Wallets::allWithUsername()
-            ->orderBy('balance')
-            ->each(fn ($wallet) => CacheVoterCountByPublicKey::dispatch($wallet->public_key)->onQueue('voters'));
+        $walletCache = new WalletCache();
+
+        $select = [
+            '"wallets"."public_key"',
+            'COUNT("voters"."public_key") total',
+        ];
+
+        $results = Wallets::allWithUsername()
+            ->selectRaw(implode(', ', $select))
+            ->join(
+                'wallets as voters',
+                /* @phpstan-ignore-next-line */
+                DB::raw('"wallets"."public_key"'),
```

Comment:
```suggestion
                'wallets.public_key',
```

would the above not achieve the same? 🤔 without the need for `DB::raw`

Reply:
@ItsANameToo yep